src/utilities/sequence-editor/to-seq-json.ts

```diff
@@ -194,9 +198,9 @@ function parseActivateLoad(stepNode: SyntaxNode, text: string): Activate | Load
     metadata,
     models,
     sequence,
-    time,
+    ...(!isRTC ? { time } : {}),
     type: stepNode.name === 'Load' ? 'load' : 'activate',
-  };
+  } as Activate | Load | ImmediateActivate | ImmediateLoad;
```

Comment:
Can we use a different `type` attribute on Activate and ImmediateActivate in seq-json-schema? Same thing for Load and ImmediateLoad?

Reply:
The Schema change was kinda what @cartermak, Mike Schaffer, and I agreed on so we will have to go back to them and see if they are okay with changing this.

Reply:
Can we also split this up so it's clear what logic branch generated what type?

Reply:
Sorry for the slow reply -- I don't see an issue with amending the `type` attribute on the immediate activate and load steps to say `immediate_load` and `immediate_activate`, although it is redundant with those objects being in the `immediate_commands` array.

Reply:
To be consistent, I think you will need to change the `ImmediateFswCommand` object type as well to `type: 'ImmediateFswCommand';` 

```ts
export interface ImmediateFswCommand {
  args: Args;
  description?: Description;
  metadata?: Metadata;
  /**
   * Command stem
   */
  stem: string;
  type?: 'command'; <----- Change to ImmediateFswCommand
}
```

Reply:
@joswig new schema changes in place now. I am also using `overloaded` methods